_posts/en/newsletters/2019-04-02-newsletter.md

```diff
@@ -0,0 +1,269 @@
+  issue][Bitcoin Core #15555] for reporting feedback.
+
+## Network status
+
+- **Fee increases for fast confirmation:** after over a year of most
+  Bitcoin transactions confirming rather quickly as long as they paid a
+  feerate above the default minimum relay fee (except during [a brief
+  exceptional period][Newsletter #22]), a modest backlog has developed
+  over the previous week and raised the feerates for people who need
+  their transactions to confirm within the next several blocks.
+  Spenders willing to wait a bit longer can still receive a good deal.
+  For example, as of this writing, Bitcoin Core's fee estimator suggests
+  a fee per 1,000 vbytes of 533 µBTC for confirmation within 2 blocks
+  but only 25 µBTC for confirmation within 50 blocks---a 95% savings for
+  waiting up to an extra 8 hours.  For more information, we recommend
+  [Johoe's mempool statistics][] and [P2SH.info's fee estimate
+  tracker][].
+
+## News
+
+- **Trampoline payments for LN:** Pierre-Marie Padiou started a
+  [thread][trampoline thread] on the Lightning-Dev mailing list
+  suggesting that Alice could send a payment to Zed even if she didn't
+  know a path to his node by first sending a payment to an intermediate
+  node (Dan) and asking Dan to figure out the route the rest of the way
+  to Zed.  This would especially benefit Alice if she ran a lightweight
+  LN node that didn't attempt to keep track of the entire network.  For
+  increased privacy, Alice could use several intermediate nodes in
+  succession rather than just one (each one receiving it's own
+  instructions encrypted by Alice).  A downside described in the email
+  is that Alice could only make a rough guess about the required fees as
+  she wouldn't know the actual path, so she'd probably end up paying
+  more in fees than if she chose the route herself.
+
+- **Bitcoin Core schedules switch to default bech32 receiving
+  addresses:** since [version 0.16.0][0.16.0 segwit], Bitcoin Core's
+  built-in wallet has defaulted to generating P2SH-wrapped segwit
+  addresses when users want to receive payments.  These addresses are
+  backwards compatible with all widely-used software.  As
+  discussed in an [issue][Bitcoin Core #15560] and [the project's weekly
+  meeting][core meeting segwit], starting with Bitcoin Core 0.20
+  (expected about a year from now), Bitcoin Core will default to native
+  segwit addresses (bech32) that provide additional fee savings and
+  other benefits.  Currently, many wallets and services [already support
+  bech32 addresses][bech32 adoption], and if the Bitcoin Core project
+  sees enough additional adoption in the next six months to warrant an
+  earlier switch, it will instead default to bech32 receiving addresses
+  in Bitcoin Core 0.19.  Legacy address will continue to be provided
+  if the user requests them in the GUI or by RPC, and anyone who doesn't
+  want the update will be able to configure their default address type.
```

Comment:
Just to be clear, this is because of the existing addresstype option? (not because of a recent discussion/decision by the Core project)

Reply:
This sentence has multiple parts, so I'm not sure what you're specifically referring to, but everything is supported today.

1. If `addresstype=bech32` on master, then you can select a legacy address by unchecking the "Generate native segwit (bech32) address" box that's checked by default in the GUI (with that `addresstype` configuration).  
![2019-04-01-10_49_34_394052783](https://user-images.githubusercontent.com/61096/55337397-95ec4000-546c-11e9-885a-ffb94ac34ee6.png)

2. You can request any address type you want (P2PKH, P2SH-P2WPKH, or P2WPKH) via the second parameter of the `getnewaddressrpc` or you can generate the address you want using a descriptor or other RPC.

3. You can use `addresstype={legacy,p2sh-segwit,bech32}` as a configuration option to set your local default.

It was explicitly discussed (and, AFAICT, universally agreed) that users should continue to receive choice, so I'm imagining that it'll be done by the same basic mechanism (although maybe the GUI text will be rephrased).